docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md

```diff
@@ -21,51 +21,51 @@ Steps
+         action:
+           class_name: UpdateDataDocsAction
+   ```
 
 2. **If you only want to update certain configured Data Docs sites**:
 
-  - Add a ``site_names`` key to the ``UpdateDataDocsAction`` config.
-
-  ```yaml
-validation_operators:
-  action_list_operator: # this is a user-selected name
-    class_name: ActionListValidationOperator
-    action_list:
-    - name: store_validation_result # this is a user-selected name
-      action:
-        class_name: StoreValidationResultAction
-    - name: update_data_docs # this is a user-selected name
-      action:
-        class_name: UpdateDataDocsAction
-        site_names:
-          - team_site
-  ```
+   - Add a ``site_names`` key to the ``UpdateDataDocsAction`` config.
+
+   ```yaml
+   validation_operators:
+     action_list_operator: # this is a user-selected name
+       class_name: ActionListValidationOperator
+       action_list:
+       - name: store_validation_result # this is a user-selected name
+         action:
+           class_name: StoreValidationResultAction
+       - name: update_data_docs # this is a user-selected name
+         action:
+           class_name: UpdateDataDocsAction
+           site_names:
+             - team_site
+   ```
 
 3. **Test your configuration.**
 
    Test that your new Validation Operator Action is configured correctly:
 
-    1. Open the configuration file of a Checkpoint you created earlier and replace the value of ``validation_operator_name`` with the name of the Validation Operator you added the ``UpdateDataDocs`` action to. The details of Checkpoint configuration can be found in this [How to add validations data or suites to a Checkpoint](../../../guides/validation/checkpoints/how-to-add-validations-data-or-suites-to-a-checkpoint).
-    2. Run the Checkpoint and verify that no errors are thrown. You can run the Checkpoint from the CLI as explained in [How to run a Checkpoint in terminal](../checkpoints/how-to-run-a-checkpoint-in-terminal) or from Python, as explained in [How to run a Checkpoint in python](../checkpoints/how-to-run-a-checkpoint-in-python).
-    3. Check your configured Data Docs sites to confirm that a new Validation Result has been added.
+   1. Open the configuration file of a Checkpoint you created earlier and replace the value of ``validation_operator_name`` with the name of the Validation Operator you added the ``UpdateDataDocs`` action to. The details of Checkpoint configuration can be found in this [How to add validations data or suites to a Checkpoint](../../../guides/validation/checkpoints/how-to-add-validations-data-or-suites-to-a-checkpoint).
+   2. Run the Checkpoint and verify that no errors are thrown. You can run the Checkpoint from the CLI as explained in [How to run a Checkpoint in terminal](../checkpoints/how-to-run-a-checkpoint-in-terminal) or from Python, as explained in [How to run a Checkpoint in python](../checkpoints/how-to-run-a-checkpoint-in-python).
```

Comment:
Links seem to be broken in Netlify build?

Reply:
Please capitalize Python 🐍

Reply:
Done!  Nice catch (as it is capitalized in the linked document).